examples/survival_analysis/bayes_param_survival.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<p>This reads weird as is was written in the context of Austin's blog, maybe we can have something closer to:</p><p> </p><p>This post illustrates a parametric approach to Bayesian survival analysis in PyMC. Parametric models of survival are simpler to both implement and understand than semiparametric models; statistically, they are also more <a href="https://en.wikipedia.org/wiki/Statistical_power" target="_blank">powerful</a> than non- or semiparametric methods <em>when they are correctly specified</em>.  For an example of a <a href="https://en.wikipedia.org/wiki/Semiparametric_model" target="_blank">semiparametric</a><a href="https://en.wikipedia.org/wiki/Proportional_hazards_model#The_Cox_model" target="_blank">Cox proportional hazards model</a>, you can read this <a href="http://austinrochford.com/posts/2015-10-05-bayes-survival.html" target="_blank">blogpost</a>, but be aware that the post used and old version of PyMC and that Implementing a semiparametric model in PyMC involved some fairly complex <code>numpy</code> code and nonobvious probability theory equivalences.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>